spec/serializers/notification_serializer_spec.rb

```diff
@@ -0,0 +1,77 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe NotificationSerializer do
+  subject(:serializer) { described_class.new(notification) }
+
+  let(:notification) { create(:notification) }
+
+  describe 'move?' do
```

Comment:
should we use the convention `#` to describe we are testing a method?
`describe '#move?' do`